newrelic/resource_newrelic_nrql_alert_condition.go

```diff
@@ -191,15 +190,19 @@ func readNrqlAlertConditionStruct(condition *newrelic.AlertNrqlCondition, d *sch
 	d.Set("name", condition.Name)
 	d.Set("runbook_url", condition.RunbookURL)
 	d.Set("enabled", condition.Enabled)
-	d.Set("nrql.0.Query", condition.Nrql.Query)
-	d.Set("nrql.0.SinceValue", condition.Nrql.SinceValue)
+	d.Set("nrql", condition.Nrql)
+	d.Set("type", condition.Type)
 
 	if condition.ValueFunction == "" {
 		d.Set("value_function", "single_value")
 	} else {
 		d.Set("value_function", condition.ValueFunction)
 	}
 
+	if err := d.Set("nrql", buildNrqlBlock(condition.Nrql)); err != nil {
```

Comment:
`flattenField` is the convention I would like to use here, so `flattenNrql` or `flattenNRQL` for this one?

If there are a lot of `flatten` and `expand` methods I thought it was nice to refactor them to a `structures_<resource_type>.go` as in the TF docs, but I'm not sure it is warranted here.

Reply:
Sounds good. I honestly wasn't sure if this fell into that category, but I'll update.